ci/build/build-code-server.sh

```diff
@@ -18,7 +18,7 @@ main() {
   fi
 
   parcel build \
-    --public-url "/static/$(git rev-parse HEAD)/dist" \
+    --public-url "." \
```

Comment:
I think this only affects the source map URL (at least for our setup; I couldn't find any other place where it was used or output in the compiled files) so now the source maps will be loaded relative to their source rather than from an absolute path. My thinking was this was a bit more flexible since if the static path changes we don't have to change it here as well. 

I think the absolute path would also cause an issue if you were serving code-server from a sub-path like `domain.tld/code-server/`.